frontend/packages/employee-frontend/src/api/attendances.ts

```diff
@@ -201,13 +201,26 @@ export async function postDeparture(
     .catch((e) => Failure.fromError(e))
 }
 
+function compareFirstNames(
+  a: JsonOfObject<AttendanceChild>,
```

Comment:
do we need `JsonOfObject` here?

Reply:
Yeah, it was giving a warning without it.

Reply:
huh, weird